tests/test_helpers.py

```diff
@@ -176,6 +176,16 @@ def get_dataset_ddo_with_access_service(client, wallet):
     return get_registered_ddo(client, wallet, metadata, service_descriptor)
 
 
+def get_dataset_ddo_with_multiple_files(client, wallet):
+    metadata = get_sample_ddo_with_multiple_files()["service"][0]["attributes"]
+    metadata["main"]["files"][0]["checksum"] = str(uuid.uuid4())
```

Comment:
Can we do a range(3) here instead of 3 separate lines?
I.e. for i in range(3): metadata["main"]["files"][i] ...